drivers/filesystems/cdfs/volinfo.c

```diff
@@ -356,7 +358,6 @@ Return Value:
 
     Buffer->TotalAllocationUnits.QuadPart = LlSectorsFromBytes( Vcb->VolumeDasdFcb->AllocationSize.QuadPart );
 
-    Buffer->AvailableAllocationUnits.QuadPart = 0;
```

Comment:
This is an imported driver so I'd rather reduce the diff here

Reply:
Thank you. I reverted it.